QKSMS/src/main/java/com/moez/QKSMS/ui/view/ComposeView.java

```diff
@@ -753,7 +755,40 @@ public void saveDraft() {
 
```

Comment:
Which line to be move to 785 ?

I don't understand : "So this way of storing the draft will only be used if there are no recipients".

Now my fix works for storing Draft SMS when Back pressed in ComposeFragment when there is at least one recipient.
If there are several recipients, my fix will save Draft for them all. I have not yet implemented the mechanism for No Recipient.

Thanks a lot!